mindsdb/api/mysql/mysql_proxy/executor/executor_commands.py

```diff
@@ -1185,6 +1200,89 @@ def answer_drop_view(self, statement):
 
         return ExecuteAnswer(answer_type=ANSWER_TYPE.OK)
 
+    def answer_create_kb(self, statement: CreateKnowledgeBase):
+        project_name = statement.name.parts[0] if len(statement.name.parts) > 1 else self.session.database
+        # get project id
+        try:
+            project = self.session.database_controller.get_project(project_name)
+        except ValueError:
+            raise SqlApiException(f'Project not found: {project_name}')
+        project_id = project.id
+
+        kb_name = statement.name.parts[-1]
+
+        # search for the model
+        # verify the model exists and get its id
+        model_identifier = statement.model
+        try:
+            model_record = self._get_model_info(
+                identifier=model_identifier,
+                except_absent=True
+            )
+
+        except PredictorRecordNotFound:
+            raise SqlApiException(f'Model not found: {model_identifier.to_string()}')
+
+        embedding_model_id = model_record['model_record'].id
+
+        # search for the vector database table
+        if len(statement.storage.parts) < 2:
+            raise SqlApiException(
+                f"Invalid vectordatabase table name: {statement.storage}"
+                "Need the form 'database_name.table_name'"
+            )
+
+        vector_db_name = statement.storage.parts[0]
+        vector_table_name = statement.storage.parts[-1]
+
+        # verify the vector database exists and get its id
+        database_records = self.session.database_controller.get_dict()
+        is_database_exist = vector_db_name in database_records
+        if not is_database_exist:
+            raise SqlApiException(f"Database not found: {vector_db_name}")
+
+        vector_database_id = database_records[vector_db_name]['id']
+
+        if statement.from_query is not None:
+            # TODO: implement this
+            raise SqlApiException("Create a knowledge base from a select is not supported yet")
+
+        params = statement.params
+
+        # create the knowledge base
+        _ = self.session.kb_controller.add(
```

Comment:
why not just:
`self.session.kb_controller.add(`

Reply:
To remind  that there is actually a return value here.